lib/const.rb

```diff
@@ -1,5 +1,5 @@
 # Defines common constants used in different parts of crew
-CREW_VERSION = '1.34.9'
+CREW_VERSION = '1.35.0'
```

Comment:
This is the correct move even if we didn't make any direct changes to the crew executable.  But we should probably clearly define reasons for bumping the version.  Should we do it for document changes and/or only for files that reside inside /usr/local/lib/crew/lib and the crew ruby script?

Reply:
I think anything inside lib being changed and also any changes to the `crew` script should trigger a version change.